### ISSUE TITLE
StatusLine and StatusLineNC share the same highlighting

### ISSUE THREAD

**Issue by omakk** [wojciechkepka/vim-github-dark#8]
As described by `:help hl-StatusLineNC`, if StatusLine and StatusLineNC share the same highlighting then Vim will use "^^^" in the status line of the current window to indicate as such. This obviously happens when using split windows.

Here's a screenshot of what that looks like:

![ghdark-issue](https://user-images.githubusercontent.com/5849730/120689966-ae07e180-c472-11eb-8447-fa01e3b325b3.png)

Indeed in `colors/ghdark.vim` we have:

```
call s:ghhl("StatusLine", "base1", "base3")
call s:ghhl("StatusLineNC", "base1", "base3")
```

I'm not sure if this is intentional, but it would be nice to have StatusLineNC set to something darker to indicate it being the non-current window. I tried `call s:ghhl("StatusLineNC", "base5", "base2")` and that looked nice to me. Here's what that looks like:

![ghdark-issue-fix](https://user-images.githubusercontent.com/5849730/120690654-7d747780-c473-11eb-9967-515b8de83df4.png)

I can submit a PR if needed, but feel free if you'd rather go with something else.

**Comment by vv9k** [comment]
Should be fixed in 30142bd 
![image](https://user-images.githubusercontent.com/46892771/137429845-6530df10-9025-4340-b740-f07416c7e551.png)

**Comment by omakk** [comment]
Thanks! Gave it a go and looks good on my end. Closing this issue.
